actool/manifest.go

```diff
@@ -422,7 +423,7 @@ func runPatchManifest(args []string) (exit int) {
 		}
 	}
 
-	err = extractManifest(tr, tw, false, newManifest)
+	err = extractManifest(tr.Reader, tw, false, newManifest)
```

Comment:
out of interest is this actually necessary? I thought it would've done the right thing if you passed `tr`